acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerRunner.java

```diff
@@ -120,6 +120,7 @@ public void shutdown() {
       params.add("--data-path");
       params.add(dataPath.toAbsolutePath().toString());
     }
+    params.add("--key-cache-limit=5");
```

Comment:
was it temporary or do you want it to make part of acceptance test?

Reply:
agreed - we really should remove cache size etc from EVERYWHERE